pynxtools/dataconverter/readers/transmission/metadata_parsers.py

```diff
@@ -40,6 +40,11 @@ def read_ref_attenuator(metadata: list) -> int:
     return int(metadata[47].split()[1].split(":")[1])
 
 
+def read_depolarizer(metadata: list) -> bool:
```

Comment:
Consider calling this function is_depolarizer_on or something. It returns a bool instead of the reading it finds.